scripts/setupTests.js

```diff
@@ -1,3 +1,11 @@
 import setupTests from '../src/setupTests'
 
+beforeEach(() => {
```

Comment:
We have to add this to Cyan's setupTests now, since it's no longer provided by default